Pod/Classes/WPMediaCollectionViewCell.m

```diff
@@ -157,13 +197,13 @@ - (void)setAsset:(id<WPMediaAsset>)asset {
     WPMediaType assetType = _asset.assetType;
     switch (assetType) {
         case WPMediaTypeImage:
+            [self displayImageAssetTypePlaceholder];
             [self fetchAssetImage];
-
             label = [NSString stringWithFormat:NSLocalizedString(@"Image, %@", @"Accessibility label for image thumbnails in the media collection view. The parameter is the creation date of the image."), formattedDate];
         break;
         case WPMediaTypeVideo:
+            [self displayVideoAssetTypePlaceholder];
             [self fetchAssetImage];
-
             label = [NSString stringWithFormat:NSLocalizedString(@"Video, %@", @"Accessibility label for video thumbnails in the media collection view. The parameter is the creation date of the video."), formattedDate];
             NSTimeInterval videoDuration = [asset duration];
```

Comment:
You have this same code in `displayVideoAssetTypePlaceholder`, so I guess it's not necessary here now?